src/main/resources/swagger/api-docs.yaml

```diff
@@ -4992,6 +5007,20 @@ components:
             additional details about the user, such as enterprise features they have access to example: '{ "enterpriseFeatures": ["feature1", "feature2"] }'
         groupMemberhipCounts:
           $ref: '#/components/schemas/GroupMembershipCounts'
+        groupsContributingToMostMemberships:
+          type: array
+          items:
+            $ref: '#/components/schemas/GroupMembershipCount'
+    GroupMembershipCount:
+      type: object
+      properties:
+        group:
+          description: unlikely to be used. Name of an internal sam group, not a managed-group which is what most users usually belong to, those will show up in policies
```

Comment:
Note this comment. Almost all google group membership is through policies but there are still some really old cases where this might show up.